src/main/java/com/gradle/enterprise/conventions/customvalueprovider/DevelocityConventions.java

```diff
@@ -29,8 +29,8 @@
 
 import static com.gradle.enterprise.conventions.customvalueprovider.ScanCustomValueNames.GIT_COMMIT_NAME;
 
-public class GradleEnterpriseConventions {
-    private static final Logger LOGGER = Logging.getLogger(GradleEnterpriseConventions.class);
+public class DevelocityConventions {
+    private static final Logger LOGGER = Logging.getLogger(DevelocityConventions.class);
     private static final String DEFAULT_GRADLE_ENTERPRISE_SERVER = "https://ge.gradle.org";
     private static final String AGREE_PUBLIC_BUILD_SCAN_TERM_OF_SERVICE = "agreePublicBuildScanTermOfService";
     private static final String GRADLE_ENTERPRISE_URL_PROPERTY_NAME = "gradle.enterprise.url";
```

Comment:
Also add a (prioritized) lookup for `gradle.develocity.url` or `develocity.url`?